test/e2e-v2/cases/clickhouse/prometheus-clickhouse-exporter/otel-collector-config.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'clickhouse-monitoring'
+          scrape_interval: 15s
+          static_configs:
+            - targets: ['clickhouse1:9363','clickhouse2:9363']
+              labels:
+                host_name: clickhouse:8123
```

Comment:
By following my question, the point is, is this process making metrics from two targets in one metric report or two?

Reply:
@CzyerChen Could you share how the metrics from clickhouse1 and clickhouse2 would be scraped here? One metric reported per target with the same host_name, or one metric for both targets in the same time?

Reply:
``` text
ClickHouseMetrics_MySQLConnection -> {SampleFamily@26369} 
"SampleFamily(samples=[Sample(name=ClickHouseMetrics_MySQLConnection, labels={net_host_port=9363, job_name=clickhouse-monitoring, 
node_identifier_host_name=clickhouse1, 
http_scheme=http, 
service_instance_id=clickhouse1:9363,
 host_name=clickhouse:8123}, 
value=0.0, timestamp=1709519785903)], context=SampleFamily.RunningContext(metricName=null, meterSamples={}, defaultHistogramBucketUnit=SECONDS))"
```

The OTEL will scrape and send the metrics node by node, in other words, OAP only received 1 node's metric at one time, so we can't sum the metrics from separate node's metrics.

Reply:
Have to make it from two targets to two hostnames. Prometheus endpoint fetches data for per node.
I will correct `otel-collector-config.yaml`.